### ISSUE TITLE
[BUG]System.AggregateException: One or more errors occurred. (Deserialised application settings is null!)  [Inner Exception] System.Exception: Deserialised application settings is null!

### ISSUE THREAD

**Issue by Ivy-Cyber** [pizzaboxer/bloxstrap#2141]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

System.AggregateException: One or more errors occurred. (Deserialised application settings is null!)

[Inner Exception]
System.Exception: Deserialised application settings is null!

**Comment by xsubbiex** [comment]
I uninstalled Bloxstrap and removed both Bloxstrap and Roblox files from %temp%. I restarted and redownloaded Bloxstrap and still got the same error message. However, when I tried again, reinstalling Bloxstrap as an admin, it gave the bug again. But on the second attempt, it downloaded successfully. Maybe if you do the same, it might work.

Summary of Troubleshooting Steps:

Run as Admin: Always run the installer as an administrator to ensure it has full system access.

Check for Software Conflicts: Temporarily disable antivirus software or other security measures to see if they are causing the issue.

Clear More Directories: In addition to %temp%, clear other temporary files and directories related to Roblox and Bloxstrap.

Update System: Ensure your operating system and all drivers are up to date.

Use a Clean Boot: Perform a clean boot to start Windows with a minimal set of drivers and startup programs to see if something is interfering with the installation.

Check Logs: Look for installation logs or error messages that might give more insight into what is causing the problem.
